test/util.js

```diff
@@ -529,8 +542,19 @@ function checkCursorPaginatedHistoryObject ({
  * @param {String}   params.filters[i].prop
  * @param {Function} [params.filters[i].customGetValue(obj)]
  *   custom getter to value with prop (by default obj[prop])
- * @param {Function} [params.filters[i].customCheck(obj, value)]
- *   if specified, will perform a custom check on that property
+ *
+ * Custom check filter functions, one by filter type
+ * `getValue` is the getter function (obj[prop] or customGetValue(obj))
+ * @param {Function} [params.filters[i].customExactValueFilterCheck(obj, value)]
+ *  where value is a primitive type
+ *  default: (obj, value) => getValue(obj) === exactValue
+ * @param {Function} [params.filters[i].customArrayFilterCheck(obj, values)]
+ *  where values is an array
+ *  default: (obj, value) => values.includes(getValue(obj))
+ * @param {Function} [params.filters[i].customRangeFilterCheck(obj, rangeValues)]
+ *  where rangeValues is an object with the following properties: `gte` and `lte`
+ *  default: rangeValues.gte <= getValue(obj) && getValue(obj) <= rangeValues.lte
```

Comment:
`gt` and `lte` are supported in some endpoints but not here?

Reply:
That's right. I tested only `gte` and `lte` operators but not all combinations.
I'll add them too to cover all cases, thanks.